client/src/Tools/_framework/ToolPanels/MoveContentToFolder.tsx

```diff
@@ -0,0 +1,194 @@
+          return -1;
+        } else if (b.isFolder && !a.isFolder) {
+          return 1;
+        } else {
+          return 0;
+        }
+      });
+
+    let folderData;
+    if (newActiveFolderId) {
+      const { data: contentInfo } = await axios.get(
+        `/api/getMyContentInfo/${newActiveFolderId}`,
+      );
+
+      folderData = {
+        name: contentInfo.name,
+        id: newActiveFolderId,
+        parentFolderId: contentInfo.parentFolderId,
+      };
+    } else {
+      folderData = null;
+    }
+
+    const folder = setActiveView((_) => {
+      return { folder: folderData, contents: content };
+    });
+  }
+
+  const initialRef = React.useRef(null);
+
+  const fetcher = useFetcher();
+
+  return (
+    <>
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        size="xl"
+        initialFocusRef={initialRef}
+      >
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>
+            <Heading>Move "{contentName}" to folder:</Heading>
+            <HStack>
+              {activeView.folder ? (
+                <>
+                  <IconButton
+                    icon={<ArrowBackIcon />}
+                    aria-label="Back"
+                    onClick={() =>
+                      updateActiveView(activeView.folder!.parentFolderId)
+                    }
+                  />
+                  <Text>{activeView.folder!.name}</Text>
+                </>
+              ) : (
+                <Text>My Activities</Text>
+              )}
+            </HStack>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <SimpleGrid columns={1} spacing={0}>
+              {activeView.contents.length === 0 ? (
+                <Text as="i">This folder is empty.</Text>
+              ) : (
+                activeView.contents.map((content) => (
+                  <Button
+                    key={`moveContentItem${content.id}`}
+                    variant="outline"
+                    width="500px"
+                    height="2em"
+                    onClick={() => {
+                      if (content.isFolder && content.id !== id)
+                        updateActiveView(content.id);
+                    }}
+                    isDisabled={!content.isFolder || content.id === id}
+                  >
+                    <Text width="50%" marginTop="auto" marginBottom="auto">
+                      {content.name}
+                    </Text>
+                  </Button>
+                ))
+              )}
+            </SimpleGrid>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button ref={initialRef} mr={3} onClick={onClose}>
+              Cancel
+            </Button>
+            <Button
+              onClick={() => {
+                fetcher.submit(
+                  {
+                    _action: "Move",
+                    id,
+                    folderId: activeView.folder ? activeView.folder.id : null,
```

Comment:
Sending `null` here currently breaks the `Move` action, as `null` gets stringified to "null" on the post, and the `Move` action doesn't protect against this. 

The best solution is probably to add protection to the `Move` action, making it check for "null" explicitly like `Duplicate Activity` does.
```
    await axios.post(`/api/moveContent`, {
      id: formObj.id,
      desiredParentFolderId:
        formObj.folderId === "null" ? null : formObj.folderId,
      desiredPosition: formObj.desiredPosition,
    });
```